models/activities/action.go

```diff
@@ -451,16 +451,28 @@ func GetFeeds(ctx context.Context, opts GetFeedsOptions) (ActionList, int64, err
 	}
 
 	sess := db.GetEngine(ctx).Where(cond).
-		Select("`action`.*"). // this line will avoid select other joined table's columns
+		Select("`action`.id").
 		Join("INNER", "repository", "`repository`.id = `action`.repo_id")
 
 	opts.SetDefaultValues()
 	sess = db.SetSessionPagination(sess, &opts)
 
-	actions := make([]*Action, 0, opts.PageSize)
-	count, err := sess.Desc("`action`.created_unix").FindAndCount(&actions)
+	actionIDs := make([]int64, 0, opts.PageSize)
+	if err := sess.Table("action").Desc("`action`.created_unix").Find(&actionIDs); err != nil {
```

Comment:
Why not use `FindAndCount` directly here?

Reply:
Because `FindAndCount` will not generate SQL like `select count(id) from ...`.